tequilapi/endpoints/identities.go

```diff
@@ -226,14 +299,25 @@ func validateCreationRequest(createReq *identityCreationDto) (errors *validation
 	return
 }
 
+func validatePayoutInfoRequest(req *payoutInfoDto) (errors *validation.FieldErrorMap) {
+	errors = validation.NewErrorMap()
+	if req.EthAddress == nil {
+		errors.ForField("eth_address").AddError("required", "Field is required")
+	}
+	// TODO: implement validation of eth address
+	return
+}
+
 //AddRoutesForIdentities creates /identities endpoint on tequilapi service
 func AddRoutesForIdentities(
 	router *httprouter.Router,
 	idm identity.Manager,
 	signerFactory identity.SignerFactory,
+	payoutInfoRegistry PayoutInfoRegistry,
 ) {
-	idmEnd := NewIdentitiesEndpoint(idm, signerFactory)
+	idmEnd := NewIdentitiesEndpoint(idm, signerFactory, payoutInfoRegistry)
```

Comment:
Would be so nice to have this logic separate endpoint `NewPayoutEndpoint()`

Reply:
`+`